pkg/internal/cli/status.go

```diff
@@ -56,17 +66,20 @@ func StatusForLogger(l log.Logger) *Status {
 		logger:        l,
 		successFormat: " ✓ %s\n",
 		failureFormat: " ✗ %s\n",
+		warningFormat: " # %s\n",
```

Comment:
```suggestion
		warningFormat: " ⚠ %s\n",
```

Reply:
The UTF symbol for warning is cool :) , those sites are great for copy&paste: https://www.utf8icons.com/character/9888/warning-sign

Reply:
DOne